charts/nexus3/templates/statefulset.yaml

```
@@ -8,7 +8,7 @@ metadata:
     {{- include "nexus3.labels" . | nindent 4 }}
 spec:
   serviceName: {{ include "nexus3.fullname" . }}
-  replicas: 1
+  replicas: {{ .Values.replicasStatefulset }}
```

```suggestion
  replicas: {{ .Values.replicas }}
```
This should probably have some more constraints applied.